### ISSUE TITLE
sparse-decoration must stay quite if docblock is present

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#458]
This is my code:

```
[] > app
  # Checks whether it is TRUE.
  [] > is-true
    0.eq 0 > @
```

The `sparse-decoration` complains and forces me to turn this code into this:

```
[] > app
  # Checks whether it is TRUE.
  0.eq 0 > is-true
```

However, it doesn't compile, because the docblock is allowed only in front of formations. I suggest silencing `sparse-decoration` when docblock is present.

**Comment by yegor256** [comment]
@h1alexbel WDYT?

**Comment by h1alexbel** [comment]
@yegor256 we can try to map formation object line to the comment line. For instance:

```xml
<o line="4" name="is-true" pos="2">
    <o base=".eq" line="5" name="@" pos="5">
       <o base="Q.org.eolang.number" line="5" pos="4">
          <o base="Q.org.eolang.bytes">
             <o>00-00-00-00-00-00-00-00</o>
          </o>
       </o>
       <o base="Q.org.eolang.number" line="5" pos="9">
          <o base="Q.org.eolang.bytes">
             <o>00-00-00-00-00-00-00-00</o>
          </o>
       </o>
    </o>
 </o>
...
<comments>
    <comment line="4">Checks whether it is TRUE.</comment>
 </comments>
```

If sparse decoration (`@`) is located inside the formation (`is-true`), and this formation has comment, we should not complain

**Comment by maxonfjvipon** [comment]
@yegor256 that's not true (at least it shouldn't be like that)
> However, it doesn't compile, because the docblock is allowed only in front of formations

Take a look at [this](https://github.com/objectionary/eo/blob/c117356649b26ab32d2b12e6fc51c6a3a8c638cd/eo-parser/src/main/antlr4/org/eolang/parser/Eo.g4#L51). Optional comment must be supported, if it's not - please report a bug.

If comments should not be supported - also report a bug. And reconsider the issue. Do we really need it?

**Comment by yegor256** [comment]
@maxonfjvipon you're right, my mistake
